Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "MatrixAlgebraKit"
 uuid = "6c742aac-3347-4629-af66-fc926824e5e4"
 authors = ["Jutho <jutho.haegeman@ugent.be> and contributors"]
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
I think this is correct, but just keep in mind that we typically only bump the version when we actually register, so the assumption that this PR doesn't introduce breaking changes therefore this update is not breaking doesn't always hold.

In principle we've added the pullback support since v0.1.1, which is a rather substantial amount of code, which might warrant increasing the version.
Definitely curious to hear your opinion on this, the v0.x semver rules are always a bit tricky to me.

Reply:
Got it, thanks for the heads up. Would you prefer that I don't bump the version for now? This kind of thing is why I like to bias towards making releases after every PR, so there isn't a question about when to make a release and features become available right away.

I usually just follow the rule that I bump `0.x.y` to `0.x.y+1` if it isn't breaking (so that includes adding new features), and then `0.x.y` to `0.x+1.0` if it is breaking. If a package doesn't have many dependents I don't think it really matters but once there are a decent number of dependents, minimizing the number of breaking releases is obviously friendlier.

Reply:
I'm definitely okay with bumping it here, just wanted to give you the heads up because I know the workflow is a bit different :)

Reply:
Yes, definitely good to get on the same page about that!